corehq/apps/domain/views/releases.py

```diff
@@ -109,7 +109,7 @@ class ManageReleasesByAppProfile(BaseProjectSettingsView):
     page_title = ugettext_lazy("Manage Releases By App Profile")
 
     @cached_property
-    def form(self):
+    def create_form(self):
```

Comment:
Nitpick: how about `creation_form` so it doesn't sound like it does the creation? (I know it's a property, but still.)

Reply:
Hrm, I see other places in HQ use `create_form` in this way, though I still think it's confusing.

Reply:
i think `create_form` is right.
Do you think `search_form` should be renamed as well?

Reply:
FWIW I agree with Jenny on this one - I'd expect `create_form` to be a callable that returns a form, and `creation_form` to be a "noun" that can be referenced multiple times.

`search_form` sounds more like a noun than a verb to me though, so that name seems fine.

Reply:
makes sense, [updated](https://github.com/dimagi/commcare-hq/pull/25380/commits/24d999800d32306b7f21a2e589633d8c2819c1d4)